packages/fxa-auth-server/lib/metrics/glean/index.ts

```diff
@@ -307,7 +311,11 @@ export const logErrorWithGlean = ({
       glean[
         funnel as keyof Omit<
           ReturnType<typeof gleanMetrics>,
-          'resetPassword' | 'oauth' | 'thirdPartyAuth' | 'account'
+          | 'resetPassword'
+          | 'oauth'
+          | 'thirdPartyAuth'
+          | 'account'
+          | 'twoFactorAuth'
```

Comment:
Was just coming to comment that this line was missing, but you're on it 💪 
Other than this, all looked good - I've tested locally and see the event in logs!